crates/sui-benchmark/tests/simtest.rs

```diff
@@ -140,6 +143,25 @@ mod test {
         test_simulated_load(test_cluster, 120).await;
     }
 
+    #[sim_test(config = "test_config()")]
+    async fn test_simulated_load_pruning() {
+        let test_cluster = build_test_cluster(7, 0).await;
+        test_simulated_load(test_cluster.clone(), 5).await;
+        // await for all transactions to get into checkpoints
+        tokio::time::sleep(Duration::from_secs(3)).await;
```

Comment:
yes, FWIU checkpoint creation is around 3 seconds
suggestions to remove sleep statement are welcome. We do need to await all transactions to get into checkpoints, otherwise verification below would fail. There might be a couple of objects that still have old versions alive

Reply:
yes, it's due to checkpoint boundary. Will make it more clear